### PR TITLE
Update UCSF discovery form to show icon. Update substance use icon

### PR DESCRIPTION
**Pull request by schroerbrian**
Adds icon to UCSF discovery form title
image

### CODE REVIEW

File: app/pages/UcsfDiscoveryForm/UcsfDiscoveryForm.tsx [hunk 1]
```diff
@@ -52,6 +53,13 @@ const Page = () => {
   const subcategories: SubcategoryRefinement[] =
     useSubcategoriesForCategory(category?.id ?? null) || [];
 
+  const getResourceIcon = () => {
+    const selectedResource = UCSF_RESOURCES.find(
+      (c) => c.slug === selectedResourceSlug
+    );
+    return selectedResource?.icon ?? "";
+  };
```
Comment (richardxia) [inline app/pages/UcsfDiscoveryForm/UcsfDiscoveryForm.tsx:61]: Is it necessary to wrap this in a function, or can we just directly assign the value to a variable? e.g.

```js
const selectedResource = UCSF_RESOURCES.find(...);
const iconName = selectedResource?.icon ?? "";
```

**Review by richardxia — APPROVED** [review]
Not sure if you meant for this to be reviewed, but the code LGTM!

**Comment by schroerbrian** [comment]
> Not sure if you meant for this to be reviewed, but the code LGTM!

Ah thanks! Well, I was going to wait to get design feedback before bugging you, but thanks for reviewing anyhow. If Dongyeon or Joan have any requests that require significant changes, I'll re-request your review

File: app/components/ucsf/Section/Section.tsx [hunk 2]
```diff
@@ -8,7 +8,7 @@ export const Section = ({
   body,
   addClass,
 }: {
-  title?: string;
+  title?: string | JSX.Element;
```
[the remark this answers is not in the document]
Comment (schroerbrian) [reply app/components/ucsf/Section/Section.tsx]: Oh awesome, thanks. I hadn't realized this

**Pull request merged**